torch/_inductor/codegen/wrapper.py

```diff
@@ -3111,12 +3111,12 @@ def generate_kernel_call(
         ), f"expected grid to be a list or tuple but got: {grid=}"
 
         grid = [V.graph.sizevars.simplify(item) for item in grid]
-        grid_has_unbacked_symbols = any(free_unbacked_symbols(item) for item in grid)
+        grid_uses_symbolic_shapes = any(item.free_symbols for item in grid)
         grid_args = [self.grid_expr_printer(item) for item in grid]
         grid_args_str = ", ".join(grid_args)
         self.writeline(f"Grid {grid_name} = Grid({grid_args_str});")
 
-        if grid_has_unbacked_symbols:
+        if grid_uses_symbolic_shapes:
             self.writeline(f"if ({grid_name}.is_non_zero()) {{")
         kernel_var_name = f"kernels.{name}" if V.graph.aot_mode else name
         self.writeline(
```

Comment:
This is indented in the AOTInductor code. Not sure how to cleanly make this indented.

Reply:
A proper fix will need to refactor `self.lines` and `self.wrapper_call`. Fine as is for this PR.

Reply:
got it, will take a look in follow-up pr